frontend_app_kanban/src/component/Kanban.jsx

```diff
@@ -427,6 +427,7 @@ export class Kanban extends React.Component {
           </div>
           {state.editedCardInfos && (
             <CardPopup
+              overflow='auto'
               customClass={classnames('kanban__KanbanPopup', { hidden: state.boardState !== BOARD_STATE.LOADED })}
```

Comment:
Instead of mixing js with the style, I would prefer that this overflow be applied to class `.kanban__KanbanPopup .cardPopup__container`.